python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_node_type.schema.yml

```diff
@@ -1341,6 +1341,18 @@ $defs:
                 description: |-
                   Number of parallel links towards HA switches.
                   Can be used to reserve IP addresses for future parallel HA links.
+              port_channel_id:
+                type: int
+                convert_types:
+                  - str
+                description: |-
+                  Port-channel ID to use when using multiple interfaces for direct HA.
```

Comment:
You are also using this for a single interface right?

Reply:
fixed